services/core-api/app/api/incidents/models/mine_incident.py

```diff
@@ -310,7 +310,7 @@ def send_incidents_email(self):
                 "mine_name": self.mine_table.mine_name,
                 "mine_no": self.mine_table.mine_no,
             },
-            "minespace_incident_link": f'{Config.MINESPACE_PRODUCTION_URL}/mines/{self.mine.mine_guid}/incidents/{self.mine_incident_guid}',
```

Comment:
There were a bunch of other places that had links setup with production urls, I wasn't too sure if they need to be adjusted so I left them alone for now.